libkbfs/kbpki_client.go

```diff
@@ -65,6 +65,25 @@ func (k *KBPKIClient) ResolveImplicitTeam(
 		ctx, assertions, suffix, tlfType, false, "")
 }
 
+// ResolveImplicitTeamByID implements the KBPKI interface for KBPKIClient.
+func (k *KBPKIClient) ResolveImplicitTeamByID(
+	ctx context.Context, teamID keybase1.TeamID, tlfType tlf.Type) (
+	ImplicitTeamInfo, error) {
+	name, err := k.serviceOwner.KeybaseService().ResolveImplicitTeamByID(
+		ctx, teamID)
+	if err != nil {
+		return ImplicitTeamInfo{}, err
+	}
+
+	assertions, suffix, err := tlf.SplitExtension(name)
+	if err != nil {
+		return ImplicitTeamInfo{}, err
+	}
+
+	return k.serviceOwner.KeybaseService().ResolveIdentifyImplicitTeam(
+		ctx, assertions, suffix, tlfType, false, "")
```

Comment:
Could we get a unified call that does these two things in one RPC?

Reply:
No, this is the way CORE wanted to do it.  It's ok.